src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -137,6 +137,12 @@ trait MarathonConf
     case None => Set(ResourceRole.Unreserved)
   }
 
+  lazy val groupRoleBehavior = opt[GroupRoleBehavior](
+    name = "group_role_behavior",
+    descr = "Defines the behavior for group roles. 'Top' means that all new child apps and pods are launched" +
```

Comment:
It seems that we set a default for top-level groups, and store this? In such a case, I think this should read "all new groups get enforced role enabled by default"; existing groups will continue to not enforce role, right?